SPECS/python3/python3.spec

```
@@ -112,10 +103,9 @@ to build python programs.
 %package        pip
 Summary:        The PyPA recommended tool for installing Python packages.
 Group:          Development/Tools
-Requires:       python3 = %{version}-%{release}
-Requires:       python3-xml = %{version}-%{release}
+Requires:       %{name} = %{version}-%{release}
 Provides:       python3dist(pip) = %{version}-%{release}
-Provides:       python3.7dist(pip) = %{version}-%{release}
+Provides:       python3.9dist(pip) = %{version}-%{release}
```

Can/Should we use `%{majmin}` here?

```suggestion
Provides:       python%{majmin}dist(pip) = %{version}-%{release}
```

---

Suggestion commit taken, thanks!